packages/angular/build/src/utils/worker-pool.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import { getCompileCacheDir } from 'node:module';
+import { Piscina } from 'piscina';
+
+export type WorkerPoolOptions = ConstructorParameters<typeof Piscina>[0];
+
+export class WorkerPool extends Piscina {
```

Comment:
Would a build function that builds a Piscina instance this way be simpler?

Reply:
It will be further abstracted in the future.